src/main/java/com/google/devtools/build/lib/skyframe/SequencedSkyframeExecutor.java

```diff
@@ -513,29 +515,40 @@ private void handleDiffsWithMissingDiffInformation(
         diffPackageRootsUnderWhichToCheck.add(pair.getFirst());
       }
 
-      EnumSet<FileType> fileTypesToCheck =
-          EnumSet.of(FileType.EXTERNAL_REPO, FileType.EXTERNAL_IN_MANAGED_DIRECTORY);
+      EnumSet<FileType> fileTypesToCheck = EnumSet.noneOf(FileType.class);
+      Iterable<SkyValueDirtinessChecker> dirtynessCheckers = ImmutableList.of();
```

Comment:
nit: I think dirt*i*nessCheckers reads better and is more consistent with the other types and variables here.